src/index.html

```diff
@@ -9,6 +9,74 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="nav">
+      <a class="logo" href="#home">
+        <img
+        src="./images/logo.png"
+        alt="moyo_icon"
+        >
+      </a>
+      <nav>
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+            class="link is-active"
+            href="#apple"
+            >apple
+            </a>
```

Comment:
```suggestion
            <a
              class="link is-active"
              href="#apple"
            >
              Apple
            </a>
```
аналогічно для інших елементів

Reply:
також якщо атрибутів тільки два - то можна залишати їх на одному рядку